web/src/compose_reply.ts

```diff
@@ -252,7 +252,6 @@ export function quote_message(opts: {
         // Setting the stream_id to undefined will automatically open the recipient popover
         compose_actions.start({
             message_type: message.type,
-            topic: "",
```

Comment:
Hmm, is this going to use the topic of the message being forwarded, or of the current conversation? I'd expect to need to pass `message.topic` here.

(What happens in the combined feed?)